README.md

```diff
@@ -416,6 +417,39 @@ See the <a href="#references">References</a> section for more details.
   mdl, _ = scorer.ensemble([mdl for _, mdl in models])
   ```
 
+</details>
+
+<details>
+  <summary>First Stage Model Selection (click to expand)</summary>
+
+First stage models can be selected either by passing in cross-validated models (e.g. `sklearn.linear_model.LassoCV`) to EconML's estimators or perform the first stage model selection outside of EconML and pass in the selected model. 
+
+```Python
+from econml.dml import LinearDML
+from sklearn import clone
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.model_selection import GridSearchCV
+
+cv_model = GridSearchCV(
+              estimator=RandomForestRegressor(),
+              param_grid={
+                  "max_depth": [3, None],
+                  "n_estimators": (10, 30, 50, 100, 200),
+                  "max_features": (2, 4, 6),
+              },
+              cv=5,
+           )
+# First stage model selection within EconML
+# This is slower, but more direct
+est = LinearDML(model_y=cv_model, model_t=cv_model)
+# First stage model selection ouside of EconML
+# Faster, but needs boilerplate code
```

Comment:
I feel we might want to take a stance and recommend this e.g.: 
as long as you are not selecting over a huge set of hyperparameter values, then hyperparameter tuning outside of econml is recommended both for statistical and computational reasons.

Reply:
done!